ecosystem/typescript/sdk/src/aptos_client.test.ts

```diff
@@ -477,3 +477,26 @@ test(
   },
   30 * 1000,
 );
+
+test(
+  "gets block by height", 
+  async () => {
+    const blockHeight = 100;
+    const client = new AptosClient(NODE_URL);
```

Comment:
@jjleng btw the test doesn't pass with `NODE_URL`. So as other tests in this file. I got the test passed with `"https://fullnode.devnet.aptoslabs.com/"`.